src/main/java/seedu/address/logic/commands/SetCommand.java

```diff
@@ -0,0 +1,80 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.LastMet;
+import seedu.address.model.person.Person;
+import seedu.address.storage.SetStorage;
+
+/**
+ * Sets the LastMet Overdue time period for all Clients.
+ */
+public class SetCommand extends Command {
+    public static final String COMMAND_WORD = "set";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": sets the LastMet overdue time period. "
+            + "Parameters: "
+            + "NUMBER_OF_DAYS "
+            + "Example: " + COMMAND_WORD + " "
+            + "90. ";
+
+    public static final String MESSAGE_SUCCESS = "LastMet Overdue time period has been set to %1$s days.";
+    /**
+     * Creates a LastMetCommand to update last mete date of the specified {@code Person}
+     */
+    private int overdueTimePeriod;
+
+    /**
+     * Creates a SetCommand to update last met overdue duration of the LastMet class
+     */
+    public SetCommand(Integer numberOfDays) {
+        requireNonNull(numberOfDays);
+
+        this.overdueTimePeriod = numberOfDays;
+    }
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        LastMet.setLastMetDuration(this.overdueTimePeriod);
+        List<Person> lastShownList = model.getFilteredPersonList();
```

Comment:
Might want to check if it is still getFilteredPersonList() after the change from the previous merge. I think this is causing the compile issue.